FirebaseVertexAI/Sample/GenerativeAITextSample/Screens/SummarizeScreen.swift

```diff
@@ -61,6 +65,10 @@ struct SummarizeScreen: View {
   }
 
   private func onSummarizeTapped() {
+    // MARK: Close Keyboard
```

Comment:
MARK comments should be used to separate parts of the code (see https://www.avanderlee.com/xcode/xcode-mark-line-comment/).

I think we don't need a comment here, as focus handling should be a well-known technique in the community.

Reply:
Removed 🙏